src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj

```diff
@@ -45,30 +45,35 @@
       <ZipItem Include="scmApplicationHost.xdt">
         <Destination>$(PackageVersion)</Destination>
       </ZipItem>
+      <!-- Tools -->
+      <ZipItem Include="$(InputBinCfgRoot)\AnyCPU\Tools\Attach\*"
+               Exclude="$(InputBinCfgRoot)\AnyCPU\Tools\Attach\*.pdb">
+        <Destination>$(PackageVersion)\Tools\Attach</Destination>
+      </ZipItem>
       <!-- Instrumentationx86Files -->
```

Comment:
Did this need to get added to the msi as well?

Reply:
Justin and I talked about this offline yesterday. 

We're foregoing that for now since it's currently not used and requires some more refactoring since attach.exe will need to interact with the proxy.

Our general design would have an "attachforwarder.exe" that calls to the specific CLRIE version of attach.exe so we can keep the API between CLRIE and attach.exe consistent.

---
In reply to: [384754757](https://github.com/microsoft/CLRInstrumentationEngine/pull/212#discussion_r384754757) [](ancestors = 384754757)

Reply:
Additionally, this executable is .NET Core 3.1, which will not work in the MSI case unless we do a self-contained publish or install .NET Core 3.1 on the machine as a dependency. Or we compile a .NET Framework version of this as well. Either way, we are holding off on making any decisions on that at this time.

---
In reply to: [384754757](https://github.com/microsoft/CLRInstrumentationEngine/pull/212#discussion_r384754757) [](ancestors = 384754757)